### PR TITLE
Mike segmentdocs

### PR DESCRIPTION
**Pull request by mcough2**
_A summary of your pull request, including the what change you're making and why._

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Comment by marinhero** [comment]
Hi Mike! Thanks for contributing to our action-destinations repo. To make the review process simpler we ask you to conform to the guidelines established on the PR template:

> A summary of your pull request, including the what change you're making and why.
Testing
Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing.

Also, would you please change the name of the PR to indicate which partner you are pushing the code from?

Thanks!

File: docs/metronome-docs.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
```
Comment (dlasky) [inline docs/metronome-docs.md:1]: i think this is meant for https://github.com/segmentio/segment-docs

Comment (mcough2) [reply docs/metronome-docs.md:1]: @dlasky - I added the markdown of the docs for the new Metronome (Actions) - metronome-doc.md. Should I submit that through a PR against your repo or against https://github.com/segmentio/segment-docs

Comment (dlasky) [reply docs/metronome-docs.md:1]: the latter is my understanding. cc @mericsson

Comment (marinhero) [reply docs/metronome-docs.md:1]: The docs repo is the right call here. There's currently no way to surface docs from this repo into our public docs.

**Pull request closed without merging**